app/javascript/components/payment-error-button.vue

```diff
@@ -32,24 +39,80 @@
 <script>
 
 import invoiceApi from '../api/invoices';
+import { mapState, mapGetters, mapActions } from 'vuex';
+import utils from '../utils/';
+import cartProducts from './cart-products.vue';
+import _ from 'lodash';
 
 const SHOW_APOLOGY_BUTTON = 10000;
-const CLOSE_MODAL_AFTER_IDLE_WAIT = 7000;
+const CLOSE_MODAL_AFTER_IDLE_WAIT = 20000000;
 
 export default {
+  components: {
+    cartProducts,
+  },
   data() {
     return {
+      bla: false,
       showApologyButton: false,
+      showDebtButton: false,
+      showNameTextBox: false,
+      showApologyText: true,
+      apologyTextTitle: 'Error en el pago',
+      apologyText: 'Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro',
     };
   },
+  computed: {
+    ...mapGetters({
+      'products': 'productsAsArray',
+      'totalAmount': 'totalAmount',
+      'totalPrice': 'totalPrice',
+      'totalFee': 'totalFee',
+    }),
+    cartProducts() {
+      return this.products.filter(product => product.amount > 0);
+    },
+  },
   methods: {
     show() {
+      console.log(this.productsArrayToModalText());
       this.$modal.show('apology-modal');
+      this.showDebtButton = true;
+      this.apologyTextTitle = 'Error en el pago',
```

Comment:
Expected an assignment or function call and instead saw an expression  no-unused-expressions<br>Unexpected use of comma operator                                       no-sequences